src/backend/linux_raw/process/syscalls.rs

```diff
@@ -328,6 +336,7 @@ pub(crate) fn setpriority_process(pid: Option<Pid>, priority: i32) -> io::Result
     }
 }
 
+#[cfg(not(target_arch = "loongarch64"))]
```

Comment:
I don't think we should deprive LoongArch from having old-style `{get,set}rlimit` API support. As you can see, this function is actually just a `prlimit64` wrapper.